docs/source/troubleshooting/troubleshooting-driver-plugins.adoc

```diff
@@ -134,6 +134,28 @@ $ sudo virsh undefine minishift
 
 In case all of this fails, you might want to xref:../getting-started/uninstalling.adoc#[uninstall Minishift] and do a fresh install of Minishift.
 
+[[Network-not-found-no-network-with-matching-name-default]]
+=== Network not found: no network with matching name 'default'
+
+At present, the KVM driver doesn't attempt to create the default network - it assumes it's already present. If you create a VM with another tool like virt-manager first it will create it (or you can create it from the command line.) Try the following to create the network
+
+[[using-command-line]]
+==== Using Commnad Line
+----
+virsh net-undefine default
+virsh net-destroy default
+virsh net-list # This will list all the networks
+virsh net-define /usr/share/libvirt/networks/default.xml
+virsh net-start default
+virsh net-autostart default
+----
+
+[[using-virt-manager]]
+==== Using virt-manager
+----
+virt-manager > edit > connection details > virtual networks > default
+----
+
```

Comment:
We might want to add that after that user can verify by running following command if the default network is created successfully or not. 
```
$ sudo virsh net-list --all
Name                 State      Autostart 
-----------------------------------------
default              active     yes
```

Reply:
Please, do not use `sudo` as this makes the command dependent of administrator/root privileges.

The usage should be:
```
$ virsh --connect qemu:///system net-list 
```

Reply:
We should be able to verify if the network exists without using `sudo`, unless we need to create it:

```
$ virsh --connect qemu:///system net-list 
```